scripts/addon/build.py

```diff
@@ -377,6 +379,20 @@ def qtquery(qmake, propname):
                 os.system(f"{lconvert} -if xlf -i {xliff_path} -o {locale_file}")
                 os.system(f"{lrelease} -idbased {locale_file}")
 
+                xlifftool_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "utils", "xlifftool.py")
```

Comment:
I understand it's beyond the scope of this, but technically you could refactor that script a bit and import it here, instead of running it in a shell and capture the output.

Reply:
I agree, that is one of the reason I opened https://mozilla-hub.atlassian.net/browse/VPN-3955